.gitignore

```diff
@@ -76,3 +78,4 @@ _build
 *.yml
 *.dot
 *.json
+!docs/**/*.json
```

Comment:
Thank you very much @rexwangcc. I'm sorry I didn't double-check whether the `.json` files are copied into the main repo.

Reply:
No worries at all, I didn't realize it either until the CI cries, CI for the win!